src/data/faq.json

```diff
@@ -579,9 +579,9 @@
                         "anchor": "new_device",
                         "active": true,
                         "textblock": [
-                            "A backup of the old random IDs is unfortunately not possible, because the Exposure Notification API of Apple/Google does not allow the export of this data. To still be informed about possible risk encounters, we recommend that you keep a close view of the risk status on the old device for at least 14 days after you install the Corona Alert app on the new device. Please note the following instructions:",
+                            "A backup of the old random IDs is unfortunately not possible, because the Exposure Notification system of Apple/Google does not allow the export of this data. To still be informed about possible risk encounters, we recommend that you keep a close view of the risk status on the old device for at least 14 days after you install the Corona-Warn-App on the new device. Please note the following instructions:",

```

Comment:
Change "Exposure Notification system" to "Exposure Notification Framework" to match German text.

Reply:
fixed. 

one problem is that it is spelled differently everywhere like in the english/german Wikipedia articles and the documentation from Google/Apple. But I don't think we want to use the "official name" GAEN (Google/Apple Exposure Notification) either. I have now settled on "Exposure Notification Framework" which should be used across the whole FAQ in german and english.

Reply:
Thanks for fixing. I think we should be okay with using Exposure Notification Framework (abb. ENF) everywhere (but maybe in Android specific entires).